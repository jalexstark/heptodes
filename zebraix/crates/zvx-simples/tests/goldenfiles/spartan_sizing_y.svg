<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="600pt" height="350pt" viewBox="0 0 600 350">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 27.273438 321.851562 L 27.273438 28.148438 L 572.726562 28.148438 L 572.726562 321.851562 Z M 27.273438 321.851562 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 27.273438 175 L 572.726562 175 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 161.539062 321.851562 L 161.539062 28.148438 M 413.285156 321.851562 L 413.285156 28.148438 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(60%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 132.167969 204.371094 L 190.910156 145.628906 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 69.230469 133.042969 L 195.105469 275.699219 M 86.015625 216.957031 L 295.804688 74.300781 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 77.621094 175 L 245.453125 175 M 92.308594 160.316406 L 260.140625 160.316406 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(60%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 413.285156 175 L 413.285156 175 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 72.03125 133.042969 C 72.03125 136.773438 66.429688 136.773438 66.429688 133.042969 C 66.429688 129.308594 72.03125 129.308594 72.03125 133.042969 Z M 197.90625 275.699219 C 197.90625 279.433594 192.304688 279.433594 192.304688 275.699219 C 192.304688 271.964844 197.90625 271.964844 197.90625 275.699219 Z M 197.90625 275.699219 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 86.015625 212.8125 L 86.015625 221.101562 M 90.15625 216.957031 L 81.871094 216.957031 Z M 295.804688 70.15625 L 295.804688 78.445312 M 299.949219 74.300781 L 291.660156 74.300781 Z M 299.949219 74.300781 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 69.230469 133.042969 L 86.015625 216.957031 M 195.105469 275.699219 L 295.804688 74.300781 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 332.171875 175 C 332.171875 178.734375 326.570312 178.734375 326.570312 175 C 326.570312 171.265625 332.171875 171.265625 332.171875 175 Z M 500.003906 175 C 500.003906 178.734375 494.402344 178.734375 494.402344 175 C 494.402344 171.265625 500.003906 171.265625 500.003906 175 Z M 500.003906 175 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 413.285156 233.792969 L 413.285156 242.082031 M 417.429688 237.9375 L 409.144531 237.9375 Z M 413.285156 107.917969 L 413.285156 116.207031 M 417.429688 112.0625 L 409.144531 112.0625 Z M 417.429688 112.0625 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 329.371094 175 L 413.285156 237.9375 M 497.203125 175 L 413.285156 112.0625 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 179.023438 160.316406 C 179.023438 164.046875 173.421875 164.046875 173.421875 160.316406 C 173.421875 156.582031 179.023438 156.582031 179.023438 160.316406 Z M 179.023438 160.316406 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 416.085938 175 C 416.085938 178.734375 410.488281 178.734375 410.488281 175 C 410.488281 171.265625 416.085938 171.265625 416.085938 175 Z M 416.085938 175 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 69.230469 133.042969 C 86.015625 216.957031 295.804688 74.300781 195.105469 275.699219 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 329.371094 175 C 413.285156 237.9375 413.285156 112.0625 497.203125 175 "/>
</svg>
