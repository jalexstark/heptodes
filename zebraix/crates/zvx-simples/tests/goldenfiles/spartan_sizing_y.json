{
  "sizing_scheme": "SquareCenter",
  "base_width": 600.0,
  "base_height": 350.0,
  "axes_range": [
    -3.2,
    -3.5,
    9.8,
    3.5
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "segments": [
            {
              "Polyline": [
                [
                  -3.2,
                  -3.5
                ],
                [
                  -3.2,
                  3.5
                ],
                [
                  9.8,
                  3.5
                ],
                [
                  9.8,
                  -3.5
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -3.2,
                  0.0
                ],
                [
                  9.8,
                  0.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  -3.5
                ],
                [
                  0.0,
                  3.5
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                6.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              -2.2,
              1.0
            ],
            [
              0.8,
              -2.4
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Plus",
          "color_choice": "YellowBrown",
          "centers": [
            [
              -1.8,
              -1.0
            ],
            [
              3.2,
              2.4
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -2.2,
                  1.0
                ],
                [
                  -1.8,
                  -1.0
                ]
              ],
              [
                [
                  0.8,
                  -2.4
                ],
                [
                  3.2,
                  2.4
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "YellowBrown"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              0.3499999999999998,
              0.35000000000000003
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -3.0,
              3.0
            ],
            "p": [
              [
                -2.2,
                1.0
              ],
              [
                -1.8,
                -1.0
              ],
              [
                3.2,
                2.4
              ],
              [
                0.8,
                -2.4
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -0.7,
                  -0.7
                ],
                [
                  0.7,
                  0.7
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "color": "Red"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -2.2,
                  1.0
                ],
                [
                  0.8,
                  -2.4
                ]
              ],
              [
                [
                  -1.8,
                  -1.0
                ],
                [
                  3.2,
                  2.4
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -2.0,
                  0.0
                ],
                [
                  2.0,
                  0.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                0.35,
                0.35
              ]
            ]
          },
          "path_choices": {
            "color": "Blue"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              4.0,
              0.0
            ],
            [
              8.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Plus",
          "color_choice": "YellowBrown",
          "centers": [
            [
              6.0,
              -1.5
            ],
            [
              6.0,
              1.5
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  4.0,
                  0.0
                ],
                [
                  6.0,
                  -1.5
                ]
              ],
              [
                [
                  8.0,
                  0.0
                ],
                [
                  6.0,
                  1.5
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "YellowBrown"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              6.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -3.0,
              3.0
            ],
            "p": [
              [
                4.0,
                0.0
              ],
              [
                6.0,
                -1.5
              ],
              [
                6.0,
                1.5
              ],
              [
                8.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  0.0
                ],
                [
                  -0.0,
                  -0.0
                ]
              ]
            ],
            "offsets": [
              [
                6.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "color": "Red"
          }
        }
      }
    }
  ]
}
